packages/blade/src/components/Tabs/_decisions/decisions.md

```diff
@@ -0,0 +1,385 @@
+      value="review"
+      leading={TransactionIcon}
+      trailing={<Counter value={99} variant="information" />}
+    >
+      Review
+    </TabItem>
+    <TabItem value="synced" leading={CheckCircleIcon}>
+      Synced
+    </TabItem>
+    <TabItem
+      value="synced-fail"
+      leading={WarningIcon}
+      trailing={<Counter value={3} variant="negative" />}
+    >
+      Synced Failed
+    </TabItem>
+    <TabItem value="excluded" leading={SlashIcon}>
+      Excluded
+    </TabItem>
+  </TabList>
+  ...
+</Tabs>
+```
+
+<img src="./example-4.png" width="380" alt="Tabs Example 2" />
+
+## Filled Variant
+
+```jsx
+<Tabs defaultValue="refunds">
+  <TabList>
+    <TabItem value="week">Week</TabItem>
+    <TabItem value="month">Month</TabItem>
+    <TabItem value="custom">Custom</TabItem>
+  </TabList>
+
+  <TabPanel value="week">
+    <WeekGraph />
+  </TabPanel>
+  <TabPanel value="month">
+    <MonthGraph />
+  </TabPanel>
+  <TabPanel value="custom">
+    <CustomGraph />
+  </TabPanel>
+</Tabs>
+```
+
+<img src="./example-2.png" width="380" alt="Tabs Example 2" />
+
+## With Tooltip
+
+```jsx
+<Tabs>
+  <TabList>
+    <TabItem value="payments">Payments</TabItem>
+    <TabItem value="refunds">
+      <Tooltip content="Return (money) in restitution, repayment, or balancing of accounts.">
+        <TooltipInteractiveWrapper>Refunds</TooltipInteractiveWrapper>
+      </Tooltip>
+    </TabItem>
+    <TabItem value="disputes">Disputes</TabItem>
+    <TabItem value="settlements">Settlements</TabItem>
+  </TabList>
+  ...
+</Tabs>
+```
+
+<img src="./example-3.png" width="380" alt="Tabs Example 3" />
+
+## Alternative APIs
+
+We [discussed](https://github.com/razorpay/blade/pull/1693#discussion_r1345677600) another potential API: 
+
+```jsx
+<Tabs>
+  <Tab title="Payments" value="" leading={} trailing={}>
+    <Text>Tab Panel Content For Payments</Text>
+  </Tab>
+  <Tab title="Refunds" value="" leading={} trailing={}>
+    <Text>Tab Panel Content For Refunds</Text>
+  </Tab>
+  <Tab title="Disputes" value="" leading={} trailing={}>
+    <Text>Tab Panel Content For Disputes</Text>
+  </Tab>
+</Tabs>
+```
+
+We decided not to go with this API because of these downsides: 
+
+#### 1. Quite tricky/hack to implement
+
+With the API that you proposed it will be quite tricky to map the JSX to the HTML structure that a typical tab requires. 
+
+A tab's HTML structure is as follows
+
+```html
+<div>
+  <div role="tablist">
+    <button role="tab">tab item 1</button>
+    <button role="tab">tab item 2</button>
+    <button role="tab">tab item 3</button>
+  </div>
+  
+  <div>
+    <section role="tabpanel"></section>
+    <section role="tabpanel"></section>
+    <section role="tabpanel"></section>
+  </div>
+</div>
+```
+
+We need to achieve the above HTML structure, and to do this with the API that you proposed will get very hacky since now we need to use React.cloneElement and save those JSX nodes in state and render them in appropriate place. 
```

Comment:
```suggestion
We need to achieve the above HTML structure, and to do this with the above API it will get very hacky since now we need to use React.cloneElement and save those JSX nodes in state and render them in appropriate place. 
```